examples/tutorials/setup_poc.ipynb

```diff
@@ -60,14 +99,23 @@
    "id": "7f877f82",
    "metadata": {},
    "source": [
-    "By default, POC mode uses a temporary workspace in /tmp/nvflare/poc.\n",
+    "##### **POC Workspace**\n",
+    "\n",
+    "By default, POC mode uses a temporary workspace in /tmp/nvflare/poc. If you are going to use default workspace, you don't need to anything. \n",
+    "\n",
     "\n",
-    "If you prefer to set a workspace in different location than the default, you can set the environment variable NVFLARE_POC_WORKSPACE \n",
+    "If you prefer to set a workspace in different location than the default, you can change the default location with different ways.  \n",
+    "\n",
+    "We have provided few ways to change the location. Prior to 2.4.0 release, you have to use environment variable. \n",
```

Comment:
suggest remove "We have provided few ways to change the location" as it's redundant